### ISSUE TITLE
add animenzb.com

### ISSUE THREAD

**Issue by tooviral** [gboudreau/sabconnectplusplus#123]
animenzb.com is a new site using fanzub template.  Please add to sabconnect++ please

**Comment by gboudreau** [comment]
0.6.17 was just send to the app store for publishing.

**Comment by tooviral** [comment]
hmm...  dosen't seem to work

**Comment by ppslim** [comment]
Due to a spelling error in my pull request, it will not work as expected.

I would appreciate feedback in this though as you can test it as follows.

- Visit animenz.com and performa search
- Press F12 to bring up the dev tools
- Select the Console tab from the dev tools
- In the console, there is a dropdown box with "<top frame>" in it, select "SABconnect++"
- At the console cursor ">" in the window, enter the text "handleAllDownloadLinks();"
- Hit enter

This should add all the appropriate link in for you and provide the support.

You can test if the "enable" / "disable" function works based on the options panel, by using the following in the console instead.

```javascript
Initialize( 'animenzb', null, function() { handleAllDownloadLinks(); };
```

**Comment by tooviral** [comment]
First time i tried it I seen the sab pic.  But wouldn't send to sabnzbd.

Tried it again but wouldn't work.

**Comment by ppslim** [comment]
Thanks,

That's expected and shows it is doing what it needs to and the associated functions can be accessed.

Again, it is down to the typo and it will work from the next release.

**Comment by gboudreau** [comment]
I just published 0.6.18 on the Chrome store; it should become available soonish.
